webapp/packages/plugin-user-profile/src/UserMenu/UserMenu.tsx

```diff
@@ -21,16 +22,19 @@ export const UserMenu = observer(function UserMenu() {
   const styles = useS(style);
   const authInfoService = useService(AuthInfoService);
   const menu = useMenu({ menu: MENU_USER_PROFILE });
+  const userInfo = authInfoService.userInfo;
```

Comment:
Seems like this pattern is not stable as the developer can easily pass authInfoService.userInfo directly to the callback fn without reassigning this first

Reply:
fixed